backdoor/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'biz.aQute.bnd.builder'
+
+description = 'AC Debugger :: Backdoor'
+dependencies {
+  compile project(':acdebugger-api')
+  compile project(':acdebugger-common')
+    compile "org.osgi:org.osgi.core:${osgi_version}"
+    compile "io.fastjson:boon:${boon_version}"
+    compile "com.google.guava:guava:${guava_version}"
+    compileOnly "org.apache.servicemix.bundles:org.apache.servicemix.bundles.jsr305:${jsr305_version}"
+}
+
+jar {
+  manifest {
```

Comment:
❓ Is there no good analog to the `maven-bundle-plugin`? Is it something we should consider writing ourselves?

Reply:
There is an official gradle plugin that I haven't really tried out in a long time as well as a few other alternatives that I'm planning on looking into. Personally I've always found bnd itself to be a bit more challenging to work with than I'd like